File: CodeSamples/php/SwaggerClient-php/lib/Model/TerminalSettings.php
<?php

namespace Swagger\Client\Model;

use \ArrayAccess;
use \Swagger\Client\ObjectSerializer;

class TerminalSettings implements ModelInterface, ArrayAccess
{
    const DISCRIMINATOR = null;

    protected static $swaggerModelName = 'TerminalSettings';

    protected static $swaggerTypes = [
        'id' => 'int',
        'terminal_id' => 'int',
        'sec_code' => 'string',
        'gateway_terminal' => 'bool',
        'consumer_credits' => 'bool',
        'dl_required' => 'bool',
        'check_verify' => 'bool',
        'identity_verify' => 'bool'
    ];

    protected static $swaggerFormats = [
        'id' => 'int32',
        'terminal_id' => 'int32',
        'sec_code' => null,
        'gateway_terminal' => null,
        'consumer_credits' => null,
        'dl_required' => null,
        'check_verify' => null,
        'identity_verify' => null
    ];

    public static function swaggerTypes()
    {
        return self::$swaggerTypes;
    }

    public static function swaggerFormats()
    {
        return self::$swaggerFormats;
    }

    protected static $attributeMap = [
        'id' => 'ID',
        'terminal_id' => 'Terminal_ID',
        'sec_code' => 'SEC_Code',
        'gateway_terminal' => 'Gateway_Terminal',
        'consumer_credits' => 'Consumer_Credits',
        'dl_required' => 'DL_Required',
        'check_verify' => 'Check_Verify',
        'identity_verify' => 'Identity_Verify'
    ];

    protected static $setters = [
        'id' => 'setId',
        'terminal_id' => 'setTerminalId',
        'sec_code' => 'setSecCode',
        'gateway_terminal' => 'setGatewayTerminal',
        'consumer_credits' => 'setConsumerCredits',
        'dl_required' => 'setDlRequired',
        'check_verify' => 'setCheckVerify',
        'identity_verify' => 'setIdentityVerify'
    ];

    protected static $getters = [
        'id' => 'getId',
        'terminal_id' => 'getTerminalId',
        'sec_code' => 'getSecCode',
        'gateway_terminal' => 'getGatewayTerminal',
        'consumer_credits' => 'getConsumerCredits',
        'dl_required' => 'getDlRequired',
        'check_verify' => 'getCheckVerify',
        'identity_verify' => 'getIdentityVerify'
    ];

    public static function attributeMap()
    {
        return self::$attributeMap;
    }

    public static function setters()
    {
        return self::$setters;
    }

    public static function getters()
    {
        return self::$getters;
    }

    public function getModelName()
    {
        return self::$swaggerModelName;
    }

    protected $container = [];

    public function __construct(array $data = null)
    {
        $this->container['id'] = isset($data['id']) ? $data['id'] : null;
        $this->container['terminal_id'] = isset($data['terminal_id']) ? $data['terminal_id'] : null;
        $this->container['sec_code'] = isset($data['sec_code']) ? $data['sec_code'] : null;
        $this->container['gateway_terminal'] = isset($data['gateway_terminal']) ? $data['gateway_terminal'] : null;
        $this->container['consumer_credits'] = isset($data['consumer_credits']) ? $data['consumer_credits'] : null;
        $this->container['dl_required'] = isset($data['dl_required']) ? $data['dl_required'] : null;
        $this->container['check_verify'] = isset($data['check_verify']) ? $data['check_verify'] : null;
        $this->container['identity_verify'] = isset($data['identity_verify']) ? $data['identity_verify'] : null;
    }

    public function listInvalidProperties()
    {
        $invalidProperties = [];

        return $invalidProperties;
    }

    public function valid()
    {
        return count($this->listInvalidProperties()) === 0;
    }

    public function getId()
    {
        return $this->container['id'];
    }

    public function setId($id)
    {
        $this->container['id'] = $id;

        return $this;
    }

    public function getTerminalId()
    {
        return $this->container['terminal_id'];
    }

    public function setTerminalId($terminal_id)
    {
        $this->container['terminal_id'] = $terminal_id;

        return $this;
    }

    public function getSecCode()
    {
        return $this->container['sec_code'];
    }

    public function setSecCode($sec_code)
    {
        $this->container['sec_code'] = $sec_code;

        return $this;
    }

    public function getGatewayTerminal()
    {
        return $this->container['gateway_terminal'];
    }

    public function setGatewayTerminal($gateway_terminal)
    {
        $this->container['gateway_terminal'] = $gateway_terminal;

        return $this;
    }

    public function getConsumerCredits()
    {
        return $this->container['consumer_credits'];
    }

    public function setConsumerCredits($consumer_credits)
    {
        $this->container['consumer_credits'] = $consumer_credits;

        return $this;
    }

    public function getDlRequired()
    {
        return $this->container['dl_required'];
    }

    public function setDlRequired($dl_required)
    {
        $this->container['dl_required'] = $dl_required;

        return $this;
    }

    public function getCheckVerify()
    {
        return $this->container['check_verify'];
    }

    public function setCheckVerify($check_verify)
    {
        $this->container['check_verify'] = $check_verify;

        return $this;
    }

    public function getIdentityVerify()
    {
        return $this->container['identity_verify'];
    }

    public function setIdentityVerify($identity_verify)
    {
        $this->container['identity_verify'] = $identity_verify;

        return $this;
    }
    public function offsetExists($offset)
    {
        return isset($this->container[$offset]);
    }

    public function offsetGet($offset)
    {
        return isset($this->container[$offset]) ? $this->container[$offset] : null;
    }

    public function offsetSet($offset, $value)
    {
        if (is_null($offset)) {
            $this->container[] = $value;
        } else {
            $this->container[$offset] = $value;
        }
    }

    public function offsetUnset($offset)
    {
        unset($this->container[$offset]);
    }

    public function __toString()
    {
        if (defined('JSON_PRETTY_PRINT')) { // use JSON pretty print
            return json_encode(
                ObjectSerializer::sanitizeForSerialization($this),
                JSON_PRETTY_PRINT
            );
        }

        return json_encode(ObjectSerializer::sanitizeForSerialization($this));
    }
}

File: CodeSamples/python/swagger_client/models/terminal_settings.py
# coding: utf-8

class TerminalSettings(object):
    swagger_types = {
        'id': 'int',
        'terminal_id': 'int',
        'sec_code': 'str',
        'gateway_terminal': 'bool',
        'consumer_credits': 'bool',
        'dl_required': 'bool',
        'check_verify': 'bool',
        'identity_verify': 'bool'
    }

    attribute_map = {
        'id': 'ID',
        'terminal_id': 'Terminal_ID',
        'sec_code': 'SEC_Code',
        'gateway_terminal': 'Gateway_Terminal',
        'consumer_credits': 'Consumer_Credits',
        'dl_required': 'DL_Required',
        'check_verify': 'Check_Verify',
        'identity_verify': 'Identity_Verify'
    }

    def __init__(self, id=None, terminal_id=None, sec_code=None, gateway_terminal=None, consumer_credits=None, dl_required=None, check_verify=None, identity_verify=None):  # noqa: E501
        """TerminalSettings - a model defined in Swagger"""  # noqa: E501
        self._id = None
        self._terminal_id = None
        self._sec_code = None
        self._gateway_terminal = None
        self._consumer_credits = None
        self._dl_required = None
        self._check_verify = None
        self._identity_verify = None
        self.discriminator = None
        if id is not None:
            self.id = id
        if terminal_id is not None:
            self.terminal_id = terminal_id
        if sec_code is not None:
            self.sec_code = sec_code
        if gateway_terminal is not None:
            self.gateway_terminal = gateway_terminal
        if consumer_credits is not None:
            self.consumer_credits = consumer_credits
        if dl_required is not None:
            self.dl_required = dl_required
        if check_verify is not None:
            self.check_verify = check_verify
        if identity_verify is not None:
            self.identity_verify = identity_verify

    @property
    def id(self):
        return self._id

    @id.setter
    def id(self, id):
        self._id = id

    @property
    def terminal_id(self):
        return self._terminal_id

    @terminal_id.setter
    def terminal_id(self, terminal_id):
        self._terminal_id = terminal_id

    @property
    def sec_code(self):
        return self._sec_code

    @sec_code.setter
    def sec_code(self, sec_code):
        self._sec_code = sec_code

    @property
    def gateway_terminal(self):
        return self._gateway_terminal

    @gateway_terminal.setter
    def gateway_terminal(self, gateway_terminal):
        self._gateway_terminal = gateway_terminal

    @property
    def consumer_credits(self):
        return self._consumer_credits

    @consumer_credits.setter
    def consumer_credits(self, consumer_credits):
        self._consumer_credits = consumer_credits

    @property
    def dl_required(self):
        return self._dl_required

    @dl_required.setter
    def dl_required(self, dl_required):
        self._dl_required = dl_required

    @property
    def check_verify(self):
        return self._check_verify

    @check_verify.setter
    def check_verify(self, check_verify):
        self._check_verify = check_verify

    @property
    def identity_verify(self):
        return self._identity_verify

    @identity_verify.setter
    def identity_verify(self, identity_verify):
        self._identity_verify = identity_verify
